packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -309,6 +309,15 @@ export class SubscriberController {
         callBack && callBack(JSON.parse(connectorEvent));
     };
 
+    /**
+     * Listener on connectors, if this changes, this listener will get triggered with the updates
```

Comment:
I know this is also how we describe other listeners but I'm not sure what we are referring to when we say _"if **this** changes"_ (the list of connectors?) or we meant _"those"_?

Not sure if that's my english or if i'm just being pedantic but it's not super clear to me.